apis/developer.go

```diff
@@ -0,0 +1,140 @@
+package apis
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/bukhavtsov/restful-app/daos"
+	"github.com/bukhavtsov/restful-app/jwt"
+	"github.com/bukhavtsov/restful-app/models"
+	"github.com/gorilla/mux"
+	"log"
+	"net/http"
+	"strconv"
+)
+
+type developerDAO interface {
+	Create(developer *models.Developer) (int64, error)
+	Read(id int64) (*models.Developer, error)
+	ReadAll() ([]*models.Developer, error)
+	Update(developer *models.Developer) (*models.Developer, error)
+	Delete(id int64) error
+}
+
+func ServeDeveloperResource(r *mux.Router) {
+	r.Handle("/developers", jwt.VerifyMiddleware(getDevelopers)).Methods("GET")
```

Comment:
VerifyMiddleware --> bad name. We should verify token/roles/permissions and not middleware